apps/cli/src/config.ts

```diff
@@ -11,6 +11,8 @@ export const GLOBAL_CONFIG_FILE = join(
 export const LOCAL_CONFIG_FILE = ".webstudio/config.json";
 export const LOCAL_DATA_FILE = ".webstudio/data.json";
 
+export const ASSETS_BASE = "/cgi/asset/";
```

Comment:
@istarkov is it ok to hardcode this? or should it be reusable from somewhere?

Reply:
At the moment this is just being used as a placeholder. Can someone point me to the URL's of the original assets. Then we can download and pass them to that location. Right now, this is just being used a `prefix` that goes under `public` folder 👍

Reply:
ah so this is your folder, not our public one, then its fine, I would have named it in a way that tells this: original-assets or just assets .... that cgi prefix made me think its a path to our cgi script

@istarkov how does one get the original files downloaded? I am guessing they aren't even available on the public domain? 

Does cli need to use http api and talk to R2 directly like we do during upload?

Reply:
http api, same as used for the builder